src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -47,6 +50,7 @@ foam.CLASS({
     { name: 'WEAK_PASSWORD_ERR', message: 'Password is weak' },
     { name: 'SUCCESS_MSG', message: 'Account successfully created' },
     { name: 'SUCCESS_MSG_TITLE', message: 'Success' },
+    { name: 'ERROR_MSG_LOGIN', message: 'There was a problem signing in to your account' }
```

Comment:
Replace "in to" with "into": https://thewritepractice.com/into-vs-in-to/#:~:text=The%20quick%20version%3A,%E2%80%9D%20use%20%E2%80%9Cin%20to.%E2%80%9D